### PR TITLE
Rename /forget_password url to /forgot_password

### PR DESCRIPTION
**Pull request by strk**

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -166,7 +166,7 @@ disable_register_prompt = Sorry, registration has been disabled. Please contact
 disable_register_mail = Sorry, Register Mail Confirmation has been disabled.
 remember_me = Remember Me
 forgot_password= Forgot Password
-forget_password = Forgot password?
+forgot_password = Forgot password?
```
Comment (lunny) [inline options/locale/locale_en-US.ini]: duplicated.

**Review by lunny — CHANGES_REQUESTED** [review]
just remove the wrong line is ok

File: options/locale/locale_en-US.ini (hunk 1 above)
Comment (strk) [reply options/locale/locale_en-US.ini]: Fixed (I don't think this is `kind/docs`, btw)

**Comment by strk** [comment]
Hold on a second, there are duplications in each and every translation :/
I'm not sure what happened. Maybe someone added a question mark after I initially made the PR, and the rebase didn't recognize it. Fixing it

**Comment by lunny** [comment]
@strk every file has a duplicated line.

**Comment by strk** [comment]
ok, figured. The `forgot_password` key is currently being used as the title for the `Forgot Password` page,
while the `forget_password` key is used for the link text.

So the new commit (replacing the others) changes `forgot_password` to `forgot_password_title` and then changes `forget_password` to `forgot_password`. Should be good now.

**Comment by bkcsoft** [comment]
This needs a big disclaimer in the changelog 😛

**Comment by strk** [comment]
@bkcsoft you seemed less concerned in https://github.com/go-gitea/gitea/pull/862#issuecomment-279224102 ? Or is this specifically for the translation keys ?

**Comment by bkcsoft** [comment]
@strk Well, people might depend on the link, so we need a changelog entry, as for the API it doesn't care 🙂

**Comment by strk** [comment]
Did anyone setup that changelog/ support mechanism ?
I'd be glad to provide a changelog within this PR if that's the case
(and if so: where is it documented?)

**Comment by lunny** [comment]
Why we need two similar strings?

**Comment by strk** [comment]
One is the link text (usually ends with a question mark: "Forgot password?")
the other is the title of the HTML page where you click the confirmation
button (thus the "_title" suffix in the new key).

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
LGTM except @bkcsoft 's opinion

**Comment by strk** [comment]
@bkcsoft I'm ok with the changelog entry, should I add it to the current CHANGELOG, drafting the new line (and fixing conflicts as we go) ? Or we could use a `[changelog:compatibility]` tag in commit logs to extract later ? Or what else do you want me to do for this to be accepted ?

**Comment by bkcsoft** [comment]
Put it in the Changelog under "Unreleased" (make a section at the top) and someone will take take of it 😉

**Comment by thibaultmeyer** [comment]
LGTM

File: cmd/web.go [hunk 2]
```diff
@@ -254,8 +254,8 @@ func runWeb(ctx *cli.Context) error {
 		m.Any("/activate", user.Activate)
 		m.Any("/activate_email", user.ActivateEmail)
 		m.Get("/email2user", user.Email2User)
-		m.Get("/forget_password", user.ForgotPasswd)
-		m.Post("/forget_password", user.ForgotPasswdPost)
+		m.Get("/forgot_password", user.ForgotPasswd)
```
Comment (bkcsoft) [inline cmd/web.go:257]: Just put a changelog somewhere (just so it can't be merged before that 😂  )

**Review by bkcsoft — CHANGES_REQUESTED** [review]

File: cmd/web.go (hunk 2 above)
Comment (strk) [reply cmd/web.go:257]: Are you @go-gitea/maintainers ok if we start dropping changelogs in files under a `pending-changelogs/` directory, to be checked and cleaned up before final release ?

Please vote with thumbs up/down on this comment :)

**Comment by strk** [comment]
@bkcsoft section added, and squash rebased against master

**Review by bkcsoft — APPROVED** [review]

File: cmd/web.go (hunk 2 above)
Comment (bkcsoft) [reply cmd/web.go:257]: @strk `changelog/unreleased` as per #1298 😉

**Pull request merged**